ckanext/canada/templates/public/package/read.html

```diff
@@ -233,7 +233,7 @@ <h4>{{ _('Contact Information') }}</h4>
 
 {%- block custom_styles %}
 {{ super() }}
-<link rel="stylesheet" href="http://open.canada.ca/sites/all/modules/custom/opendata_blog/css/opendata_blog.css" />
+<link rel="stylesheet" href="/data/static/opendata_blog.css" />
```

Comment:
Can you do it like this instead
```
<link rel="stylesheet" href="{{ h.url_for_static('/static/opendata_blog.css" />
```
that way it will work with whatever root path we have set